install_config/monitoring/modules/opting-out-of-installing-monitoring-stack.adoc

```diff
@@ -0,0 +1,10 @@
+[id='opting-out-of-installing-monitoring-stack']
```

Comment:
This module doesn't make sense in the flow that you're setting up. I'd remove it and modify the 3.x stream to add a step to the installation topic instead.

Reply:
Just removed it, 4.0 doesn't allow to opt out of monitoring.